test/test_example.py

```diff
@@ -55,6 +55,23 @@ def test_list_extracted_into_arguments(self, first_value, second_value):
     def test_dicts_extracted_into_kwargs(self, first, second, third):
         self.assertTrue(first < third < second)
 
+    @data(1, 2, 3)
+    @data(4, 5, 6)
```

Comment:
All your examples include 2 sets of `@data`. I haven't debugged your implementation to see if this is a limitation there, but it doesn't feel like it should. In that case, it would be nice to include some example case with more than 2 `@data` entries to make that explicit.